redisvl/index/index.py

```diff
@@ -650,6 +686,44 @@ def fetch(self, id: str) -> Optional[Dict[str, Any]]:
             return convert_bytes(obj[0])
         return None
 
+    def aggregate_query(
```

Comment:
Nice! I would propose we consider consolidating this under `.query()` and have it check the type of the query and call the right internal method for search/aggregate as opposed to asking the user to call the search index searching method based on the type of query it is. cc @abrookins